docs/topics/writing_templates.md

```diff
@@ -191,7 +191,7 @@ Like `pageurl`, this will try to provide a relative link if possible, but will d
 
 ## Static files (tag)
 
-Used to load anything from your static files directory. Use of this tag avoids rewriting all static paths if hosting arrangements change, as they might between development and live environments.
+Used to load anything from your static files directory. Use of this tag avoids rewriting all static paths if hosting arrangements change, as they might be between development and live environments.
```

Comment:
"they might" is referring to the verb "change" in the previous phrase, so adding "be" isn't correct. Rewording for clarity:

```suggestion
Used to load anything from your static files directory. Use of this tag avoids rewriting all static paths if hosting arrangements change, such as when moving from development to a live environment.
```